lib/msal-browser/test/interaction_client/PopupClient.spec.ts

```diff
@@ -773,6 +775,137 @@ describe("PopupClient", () => {
                 expect(e).toEqual(testError);
             }
         });
+
+        it("retries on invalid_grant error and returns successful response", async () => {
+            const testServerTokenResponse = {
```

Comment:
We can probably move all the pre-populated test responses to TestUtils.

Reply:
Interesting. I will check if the test responses are the same across flows and move to StringConstants if they are repeatable.